core/server/data/migrations/versions/2.17/2-canonical-post.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird'),
```

Comment:
> 2-canonical-post.js

add-canonical-url-column.js?

The string lands in the migration database.

Not super important though. Just noticed :)

Reply:
Yes, renaming it sounds good :+1: Was thinking about it for a bit, but followed the same convention as in `1-og-twitter-post.js`. Renamed, so we follow better convention in the future :)